config/mainnet-addresses.json

```diff
@@ -110,6 +111,14 @@
   "PIP_RENBTC": "0xf185d0682d50819263941e5f4EacC763CC5C6C42",
   "MCD_JOIN_RENBTC_A": "0xFD5608515A47C37afbA68960c1916b79af9491D0",
   "MCD_FLIP_RENBTC_A": "0x30BC6eBC27372e50606880a36B279240c0bA0758",
+  "AAVE": "0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9",
+  "PIP_AAVE": "0x8Df8f06DC2dE0434db40dcBb32a82A104218754c",
+  "MCD_JOIN_AAVE_A": "0x24e459F61cEAa7b1cE70Dbaea938940A7c5aD46e",
+  "MCD_FLIP_AAVE_A": "0x16e1b844094c885a37509a8f76c533B5fbFED13a",
+  "UNIV2DAIETH": "0xA478c2975Ab1Ea89e8196811F51A7B7Ade33eB11",
+  "PIP_UNIV2DAIETH": "0x87ecBd742cEB40928E6cDE77B2f0b5CFa3342A09",
+  "MCD_JOIN_UNIV2DAIETH_A": "0x2502F65D77cA13f183850b5f9272270454094A08",
+  "MCD_FLIP_UNIV2DAIETH_A": "0x57dfd99f45747DD55C1c432Db4aEa07FBd5d2B5c",
```

Comment:
Confirmed the mainnet addresses with https://changelog.makerdao.com/releases/mainnet/1.2.2/contracts.json.